basic/boundCheck5.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2016, 2017 National University of Singapore
+ *
+ * This program is for testing memory bounds check interpolation: it
+ * should have no interpolations, since KLEE can check that the target is safe 
```

Comment:
@rasoolmaghareh Perhaps "interpolations" here should be "subsumptions" and "KLEE" should be "Tracer-X" or "Tracer-X KLEE"?